app/jobs/create_crm_page_job.rb

```diff
@@ -0,0 +1,58 @@
+class CreateCrmPageJob < ApplicationJob
+  queue_as :default
+
+  # Base de données "Activation"
+  # ID récupéré dans l’URL de la page
+  # Cf: https://developers.notion.com/reference/retrieve-a-database
+  NOTION_DATABASE_ID = ENV["NOTION_DATABASE_ID"].freeze
+
+  def perform(territory_id)
+    return unless ENV["NOTION_API_SECRET"]
+
+    territory = Territory.find(territory_id)
+    client.create_page(
+      parent: {
+        type: "database_id",
+        database_id: NOTION_DATABASE_ID,
+      },
+      properties: {
+        "Project name": {
+          title: [
+            {
+              text: {
+                content: territory.name.presence || territory.organisations.first&.name,
```

Comment:
On pourrait réutiliser cette méthode
```suggestion
                content: territory.name_in_stats,
```